chainer/training/trainer.py

```diff
@@ -39,10 +39,11 @@ class Trainer(object):
       should be called or not.
 
     Extensions are callable objects that take the trainer object as the
-    argument. There are two ways to define custom extensions: inheriting the
-    :class:`Extension` class, and decorating functions by
-    :func:`make_extension`. See :class:`Extension` for more details on custom
-    extensions.
+    argument. There are three ways to define custom extensions: inheriting the
+    :class:`Extension` class, decorating functions by :func:`make_extension`,
+    and defining any callable including lambda functions. The default
+    configurations of an extension is used in the former two cases. See
```

Comment:
I agree that it is not complete. How about removing the sentence `The default configurations of an extension is used in the former two cases.` because the complete explanation is already written in the class `Extension`?